src/lib/viewers/text/CSVViewer.js

```diff
@@ -53,43 +52,53 @@ class CSVViewer extends TextBaseViewer {
     load() {
         super.load();
 
-        const { representation, location } = this.options;
+        const {
+            file: { extension },
+            representation,
+        } = this.options;
         const template = representation.content.url_template;
-        const assetUrlCreator = createAssetUrlCreator(location);
-        const papaWorkerUrl = assetUrlCreator(JS[0]);
 
         return Promise.all([this.loadAssets(JS), this.getRepStatus().getPromise()])
             .then(() => {
-                this.api.get(papaWorkerUrl, { type: 'blob' }).then(papaWorkerBlob => {
```

Comment:
Papaparse 5.0.0 added the `worker: true` option as the way to load an inline worker

Reply:
How does it determine the asset url for the worker? Does it load it via a path relative to the main file?

Reply:
It appears to load the source code inline: https://github.com/mholt/PapaParse/blob/master/papaparse.js#L52